### DISCUSSION TITLE
Custom fonts for editor, and display in general

### DISCUSSION THREAD

**Discussion by crankedguy** [requarks/wiki#6639] [Help / Questions] [answered]
Hello,

I wanted to kindly ask why 
1. something so basic like adding custom fonts instead of the ugly available fonts is not implemented in such a system
2. not a single question on fonts is answered by someone responsible
3. I managed to use a custom font in general relatively easy through CSS injection, which is ok if you want to do it for general CSS elements on the whole page, and otherwise becomes a complete burden because you are waisting your time styling instead of writing content

It would be really nice if one would answer that actually.
Many thanks

**Comment by NGPixel** [comment, marked as answer]
1. Roboto is one of the most used fonts on the web. You're most likely the first person to complain about the current fonts being ugly. If it's that important to you, you can override it via CSS injection. This is literally what it's for.
2. Which questions?
3. Again, the overwhelming majority of users have no problem with the default font. If writing content is your ultimate goal, then using a font like Roboto, which is accessible, very readable and complete in almost all character sets really shouldn't bother you.

You'll be able to select from a list of predefined fonts in v3.x. And if you don't like any of them, you can again use CSS injection to use your own.

**Reply by crankedguy** [reply]
Thanks for getting back to me on that. I found simply quite a few questions regarding fonts that were all not answered. It was not meant as an offense, I just wanted to know the reason because it can't be a problem to add custom downloaded fonts to the editor list.
And ugly might be the wrong word, was also no offense. There are just literally way nicer fonts out there, and if you use wiki for more than just tech doc content the possibility would have been nice. Fonts are part of branding, too.
But whatever, I will keep doing it by CSS injections then or maybe find another solution.
Thanks!

**Reply by reysad568** [reply]
@crankedguy 
Do you get any solution for it?
I want to add some custom fonts to the font-family toolbar of editor.
any suggestion is appreciated.
